dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/formatting2/XtxtUMLFormatter.xtend

```diff
@@ -137,8 +138,14 @@ class XtxtUMLFormatter extends XbaseFormatter {
 		}
 	}
 
+	def dispatch void format(TUModifiers it, extension IFormattableDocument document) {
```

Comment:
What is your proposal then? To write a comment or to use the offset to determine the order of the regions. The latter seems a bit messy for me.